src/pages/Graph/SummaryPanelEdge.tsx

```diff
@@ -262,6 +263,11 @@ export default class SummaryPanelEdge extends React.Component<SummaryPanelPropTy
           destMetricType,
           sourceData
         );
+        // TCP (uses slightly different reporting)
+        useDest = sourceData.nodeType === NodeType.UNKNOWN;
+        useDest = useDest || this.props.namespace === 'istio-system';
```

Comment:
I'm pretty sure we have a variable or something that gives us the istio-system namesapce so we do not hardcode it. There is a way in kiali that the user can actually configure it for a different name - so we need to make sure we rely on that. See: https://github.com/kiali/kiali/blob/master/config/config.go#L161 - I'm sure the UI is getting this value and storing it somewhere that we can use rather than hardcoding istio-system.

Reply:
I agree it's a problem but afaik we don't have that config available yet in the UI.